plugins/blocks/browse/block.tpl

```diff
@@ -28,7 +28,7 @@
 				</li>
 			{/if}
 
-			{if $browseCategories}
+			{if $browseCategories->count}
```

Comment:
Oh, ok. Have no problem using object factories in templates, I would just call it browseCategoryFactory, to be clear...